cpp/daal/src/threading/threading.cpp

```diff
@@ -61,6 +61,31 @@ DAAL_EXPORT void _daal_tbb_task_scheduler_free(void *& globalControl)
     }
 }
 
+DAAL_EXPORT void _daal_tbb_task_scheduler_handle_free(void *& schedulerHandle)
+{
+    // #if defined(TARGET_X86_64)
```

Comment:
Please add some descriptive note why this change is necessary. For example:
```suggestion
    // Note: TBB 13 deletes task_scheduler_handle itself during the destruction of thread context

    // #if defined(TARGET_X86_64)
```